src/video/gl/gl_program.hpp

```diff
@@ -34,20 +34,44 @@ class GLProgram final
 
   GLuint get_handle() const { return m_program; }
 
-  GLint get_attrib_location(const char* name) const;
-  GLint get_uniform_location(const char* name) const;
+  GLint get_backbuffer_location() const { return check_valid(m_backbuffer_location, "backbuffer"); }
+  GLint get_fragcoord2uv_location() const { return check_valid(m_fragcoord2uv_location, "fragcoord2uv"); }
+  GLint get_diffuse_texture_location() const { return check_valid(m_diffuse_texture_location, "diffuse_texture_"); }
```

Comment:
Is the underscore after `diffuse_texture` a typo or not?

Reply:
It is a typo; fixed.